### PR TITLE
Clean up README links.

### LINKED ISSUE #409: The readme has dead links in it
The readme for this project contains dead links. For example, it contains a link to http://www.projectcalico.org/download/, which does not exist any more. We should read through the README, and make sure that it correctly references the right docs.

### PR DESCRIPTION
**Pull request by Lukasa**
This replaces all dead links in the README with links that actually work.

Resolves #409.

@plwhite, as you raised the original issue can you please review?

### CODE REVIEW

**Comment by matthewdupre** [comment]
doc/CalicoUTs.md still looks like a dead link (even with the docs branch merged)

Other links look fine.

File: README.md [hunk 1]
```diff
@@ -27,24 +27,24 @@ Calico integrates seamlessly with the cloud orchestration system (such as
...
 Technical documentation is [here](http://docs.projectcalico.org/);
 if you are going to contribute to the project, you'll also need to run the
-[tests](doc/CalicoUTs.md).
+tests.
 
 ## How can I get support for Project Calico?
 
 There are two options for getting support for Calico. You can simply
-[ask the community](http://www.projectcalico.org/community/) any question you
+[get in contact](http://www.projectcalico.org/contact/) any question you
```
Comment (plwhite) [inline README.md]: "get in contact any question you like" - what? I think you are missing an "and".

File: README.md [hunk 2]
```diff
@@ -27,24 +27,24 @@ Calico integrates seamlessly with the cloud orchestration system (such as
...
 To get started, you first need a working installation of
 [OpenStack](http://www.openstack.org/). Then download and install the latest
 stable build of Calico following the instructions
-[here](http://www.projectcalico.org/download/).
+[here](http://docs.projectcalico.org/en/latest/openstack.html).
 
 Technical documentation is [here](http://docs.projectcalico.org/);
 if you are going to contribute to the project, you'll also need to run the
-[tests](doc/CalicoUTs.md).
+tests.
```
Comment (plwhite) [inline README.md]: Should we have some kind of link to where the tests are? If not, we should probably not mention them here.

**Comment by plwhite** [comment]
Final comment - "how do I hack on project calico". Should that be "contribute to"? For me, hack has pretty negative connotations (though I'm willing to be told that I'm just out of date!).

Back to you to resolve the minor stuff above.

**Comment by neiljerram** [comment]
Hm, not sure. I have a personal bias towards supporting / reclaiming its good meaning. But you could be right that some of our partners might misunderstand it.

It was used positively to me once or twice yesterday, here in Cologne.

File: README.md (hunk 1 above)
Comment (ed-harrison) [inline README.md]: I would probably say something like "get in touch (see our [contact page] for the various methods) and ask any..."

**Comment by Lukasa** [comment]
I'm happy with 'hack' as well.

**Comment by Lukasa** [comment]
Thanks for your (admittedly late) feedback all, see #433 for the fruits of your labours!

**Pull request merged**